libs/core/src/lib/select/option/option.component.ts

```diff
@@ -29,9 +68,13 @@ let optionUniqueId: number = 0;
         'class': 'fd-list__item',
         '[attr.aria-disabled]': 'disabled',
         '[attr.aria-selected]': 'selected',
+        '[class.is-hover]': 'active',
```

Comment:
Shouldn't we add `is-focus` instead of hover?

Reply:
@JKMarkowski This is something that should be comming from styles. You dont want to re-define classes in your component. I would like is-`active`

Reply:
I will mark this as resolved as I work only with avaialbe classes to make it active. We should have this active style in the styles, then we can replace it.